pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -82,7 +82,7 @@ stdenv.mkDerivation rec {
   ++ optionals (!withScripting) [
     "-DKICAD_SCRIPTING_WXPYTHON=OFF"
   ]
-  ++ optional (withNgspice) "-DKICAD_SPICE=ON"
+  ++ optional (!withNgspice) "-DKICAD_SPICE=OFF"
```

Comment:
An opportunity to just remove an option that never worked.

Reply:
why would i do that if i can make it work instead?

Reply:
There is no point to keep so many knobs that are not tested. I think we can give our users some credit, to say that they can figure out how to fine tune kicad with `overrideAttrs` if they need to.